category_test.go

```diff
@@ -27,13 +27,13 @@ func TestCreateRubricCategory(t *testing.T) {
 func TestGetRubricCategory(t *testing.T) {
 	// Arrange
 	testRequest := autopilot.NewTestRequest(
-		`query CategoryGet($id:ID!){account{category(id: $id){id,name}}}`,
+		`query CategoryGet($id:String!){account{category(id: $id){id,name}}}`,
```

Comment:
This is not a good change - we are changing the contract and i'm 99% this won't actually work in the API because its expected to be an ID type

Reply:
@rocktavious you are correct and I have reverted this change. 

I originally changed this so that we can use `string` everywhere. While that's helpful if the graphql query supports an `ID` or an `Alias`, I realize now that it is not helpful for developers using our library in the case where the query ONLY accepts an `ID`. 

I have added logic to the linter part of `task parser` to acknowledge there's a situation where a `delete()` function can have args `string|ID`.